$NetBSD: patch-CMakeLists.txt,v 1.1 2022/10/06 01:30:02 markd Exp $

Not needed by pkgsrc and RUNTIME_DEPENDENCY_SET is not defined for all
platforms

--- CMakeLists.txt.orig	2022-08-17 11:35:47.000000000 +0000
+++ CMakeLists.txt
@@ -18,4 +18,4 @@ if(DO_BOOTSTRAP)
 endif()
 
 # A dummy install part to fulfill needs of kdesrc-build and possible make installs
-install(RUNTIME_DEPENDENCY_SET kapidox OPTIONAL)
+#install(RUNTIME_DEPENDENCY_SET kapidox OPTIONAL)
